tests/data/Upgrade/Infrastructure/Processor/Strategy/Composer/Steps/FooStep.php

```diff
@@ -7,15 +7,15 @@
 
 namespace UpgradeData\Infrastructure\Processor\Strategy\Composer\Steps;
 
-use Upgrade\Infrastructure\Dto\Step\StepsExecutionDto;
-use Upgrade\Infrastructure\Processor\Strategy\StepInterface;
+use Upgrade\Application\Dto\StepsExecutionDto;
```

Comment:
Please create a bug ticket for tests?

Reply:
Created a task https://spryker.atlassian.net/browse/PPLUS-1302
Bug should get steps to reproduce and other description